datahub-web-react/src/app/identity/ManageIdentities.tsx

```diff
@@ -8,6 +8,11 @@ import { UserList } from './user/UserList';
 const PageContainer = styled.div`
     padding-top: 20px;
     width: 100%;
+    max-height: 100%;
```

Comment:
why is this max height necessary?

Reply:
(why not just height)

Reply:
it doesn't seem necessary in other places

Reply:
I have kept max-height because if there is lots of data then container can take only 100% height but yes flex doing the same thing so we can remove height.